### PR TITLE
Added video-embed option to the Menu 

### LINKED ISSUE #565: Add video-embed option to the Menu
- Add video embed option to the Fixed and Embed menu types.
- Revamp the video embed UI to show a Modal instead of an alert.

@gaagul _a

### PR DESCRIPTION
**Pull request by gaagul**
Fixes #565 

**Description**

- Changed: video embed UI to show a Modal instead of an alert.
- Added: video embed option to the menu.

**Checklist**

- [x] I have added the necessary label (patch/minor/major - If package publish is required).

**Reviewers**

@AbhayVAshokan _a

### CODE REVIEW

**Comment by gaagul** [comment]
@gaagul _a

**Comment by gaagul** [comment]
@AbhayVAshokan _a

File: lib/components/Editor/EmbedOption/index.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+      onClose={handleClose}
+    >
+      <Modal.Header>
+        <Typography id="dialog1Title" style="h2">
+          Embed Video
+        </Typography>
+      </Modal.Header>
+      <Modal.Body className="space-y-2">
+        <Input
+          error={error ? "Please enter a valid URL" : null}
```
Comment (AbhayVAshokan) [inline lib/components/Editor/EmbedOption/index.jsx:63]:
```suggestion
          error={error && "Please enter a valid URL" }
```

**Review by AbhayVAshokan — CHANGES_REQUESTED** [review]
@gaagul _a

**Comment by gaagul** [comment]
@AbhayVAshokan _a Please review the changes.

**Review by AbhayVAshokan — APPROVED** [review]
Nicely done 👌

**Pull request merged**